rendering.js

```diff
@@ -64,14 +64,16 @@ function renderLivesRemaining({ livesRemaining }) {
 }
 
 function renderFallingObjects({ fallingObjects }) {
-    const fallingObjectsContainer = document.getElementById(
-        EL_IDS.fallingObjectsList,
+    const fallingObjectColumnEls = document.querySelectorAll(
+        EL_IDS.fallingObjectColumn,
     )
 
     fallingObjects.forEach((fallingObject) => {
-        const fallingObjectEl = document.createElement('li')
-        const numeratorEl = document.createElement('div')
-        const denominatorEl = document.createElement('div')
+        const fallingObjectColumnEl =
+            fallingObjectColumnEls[fallingObject.columnIndex]
```

Comment:
Does this mean that only one falling object can be in a single column at any time? 

If so, I think that's cool for now, but maybe important to note the limitation.

Reply:
No, it doesn't. You can add as much fallingObject as you want to a column. Multiple `fallingObject` can have the same `columnIndex`